src/chrome/content/rules/GoNewSoft.com.xml

```diff
@@ -1,7 +0,0 @@
-<ruleset name="GoNewSoft.com">
-        <target host="gonewsoft.com" />
-        <target host="www.gonewsoft.com" />
```

Comment:
Hello
I do not know what to do
Please mute your removal request
Or exactly tell me what to